BLSQ/iaso
# IA-2757: Review change proposals: have some data to show in the view so the users can experience the feature

Explain what problem this PR is resolving

Related JIRA tickets : [IA-2757](https://bluesquare.atlassian.net/browse/IA-2757)

## Self proofreading checklist


## Changes

Created changes request proposal review 

## How to test

From setuper folder, launch `python3 setuper.py` to create data and open review changes proposal page: [http://localhost:8081/dashboard/orgunits/changeRequest/](http://localhost:8081/dashboard/orgunits/changeRequest)

You should see differents rows with differents status(New, Approved and Rejected). 


## Print screen / video

https://github.com/BLSQ/iaso/assets/6383219/d2f9a657-5a40-469f-971c-a379995e93fa





[IA-2757]: https://bluesquare.atlassian.net/browse/IA-2757?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
